spec/requests/user_request_spec.rb

```diff
@@ -10,12 +10,20 @@
     let(:v0_user_request_headers) { {} }
     let(:edipi) { '1005127153' }
 
+    let(:cassettes_good_vet_status) do
+      [
+        { name: 'va_profile/demographics/demographics' },
+        { name: 'va_profile/veteran_status/va_profile_veteran_status_200',
+          options: { match_requests_on: %i[method body] } }
+      ]
+    end
+
     before do
       allow_any_instance_of(MHVAccountTypeService).to receive(:mhv_account_type).and_return('Premium')
       create(:account, idme_uuid: mhv_user.uuid)
       sign_in_as(mhv_user)
       allow_any_instance_of(User).to receive(:edipi).and_return(edipi)
-      VCR.use_cassette('va_profile/veteran_status/va_profile_veteran_status_200', allow_playback_repeats: true) do
+      VCR.use_cassettes(cassettes_good_vet_status) do
```

Comment:
Why the change to the cassette name here?

Reply:
I would also like to keep the directory nesting and naming scheme if possible

Reply:
That cassette is still being used, but we are not adding the demographics cassette too. So, two cassettes are being used.

Reply:
Directory nesting is also kept. See like 15 and 16 for the cassettes being used here.

Reply:
Oh! I'm sorry! I overlooked that. I see the definition above